test/testdata/schema/ghc912/vscode-extension-schema.golden.json

```diff
@@ -988,5 +1024,11 @@
         "description": "Enables semanticTokens plugin",
         "scope": "resource",
         "type": "boolean"
+    },
+    "haskell.plugin.stan.globalOn": {
+        "default": false,
+        "description": "Enables stan plugin",
+        "scope": "resource",
+        "type": "boolean"
     }
```

Comment:
A lot of the changes in this file have nothing to do with Ormolu or Fourmolu. How has that happened?

Reply:
Initially, I intended this PR to enable the stan, hlint, fourmolu, and ormolu plugins. However, hlint wasn't going smoothly and I decided to only enable fourmolu and ormolu, but I have a couple of unstaged changes on my branch. Then I accepted these test changes because I forgot I enabled too many plugins :)

Reply:
In other words, my lack of branch discipline came back to bite me, should be fixed now!